projects/user-telemetry-client/src/ITelemetryData.ts

```diff
@@ -0,0 +1,35 @@
+export interface ITelemetryEvent {
+  /**
+   * The application sending the event
+   */
+  app: string;
+
+  /**
+   * The application stage, e.g. 'CODE' | 'PROD'
+   */
+  stage: string;
```

Comment:
We can take this opportunity to use the interface already defined in this library in `definitions`. This definition supplants what's already there.

Reply:
This may be awkward to compile.

Reply:
Let's find out! If that proves to be the case, I'm sure we can figure out a way of making it less so.